### PR TITLE
Restructures readme

### PR DESCRIPTION
**Pull request by bravo-kernel**
I rewrote the readme following my notes when I implemented, this should now be (at least) the logical order to help users on their way. Consider this a concept, feedback welcome.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,61 +5,103 @@
 [![Total Downloads](https://img.shields.io/packagist/dt/muffin/obfuscate.svg?style=flat-square)](https://packagist.org/packages/muffin/obfuscate)
 [![License](https://img.shields.io/badge/license-MIT-blue.svg?style=flat-square)](LICENSE)
 
-CakePHP 3 support for ID obfuscation using any combination of HashIds, Optimus and/or Tiny.
+Primary key obfuscation for CakePHP 3 using any combination of
```
Comment (ADmad) [inline README.md]: I am not too sure about using the word "combination". Most people would only use either one of them.

Comment (bravo-kernel) [inline README.md]: As far as this heading text is concerned I could change it to something more readable like "using X, Y and/or custom strategies".

In general, it's exactly the fact that you can mix and match as needed that makes this plugin awesome and.. is actually how I use it. Optimus for user-ids, Tiny for "linkable" resources.

Sent from my Mobile Device

Comment (ADmad) [inline README.md]: Yeah i would prefer a statement with "and/or" or "one or more".

**Comment by jadb** [comment]
Nice work @bravo-kernel ! Apart from @ADmad's comment, LGTM.

**Comment by bravo-kernel** [comment]
Thanks guys, I rebased the smarter intro.

**Comment by bravo-kernel** [comment]
I have a PR for awesome-cakephp ready to once this is merged.

**Pull request merged**